packages/vitest/src/utils/index.ts

```diff
@@ -109,3 +111,20 @@ export function objectAttr(source: any, path: string, defaultValue = undefined)
   }
   return result
 }
+
+// The server needs to be running for communication
+export function shouldKeepServer(config?: ResolvedConfig) {
+  return config?.watch || config?.browser
+}
+
+export function isForSafari(config?: ResolvedConfig | CliOptions) {
```

Comment:
As I said before, create a provide/dependency injection for browser handling, so it can be easily swapped, and keep these methods there

Reply:
Ok I agree, Is it possible to put these methods in the webdriver.ts file (and change the filename to browser.ts)?